UPGRADE.md

```diff
@@ -4,6 +4,13 @@ This document serves as a reference to upgrade your current
 BetterReflection installation if improvements, deprecations
 or backwards compatibility (BC) breakages occur.
 
+## 4.0.0
```

Comment:
Is there a reason why not to make it part of version 3?

Reply:
I just thought `3.0.0` was already released: did this in a real rush, so will amend :+1: